### DISCUSSION TITLE
Could a license be added to the metadata on pypi?

### DISCUSSION THREAD

**Discussion by kgourgou** [arogozhnikov/einops#191] [General]
Hi all, 

Opening a discussion here as it's not an issue with the software. Would it be possible to add License information in the metadata section of [einops on PyPi](https://pypi.org/project/einops/)? I'd like to introduce the software for use in my team, but I understand they need to see a license on PyPi first.

**Comment by arogozhnikov** [comment]
Hi @kgourgou,
 
I've checked that distribution archive contains a license file.

Do you mean adding this information to classifiers?

**Reply by kgourgou** [reply]
Thank you, yes, in the classifiers.

**Reply by arogozhnikov** [reply]
ok, should appear with the next release
